mitmproxy/io/db.py

```diff
@@ -0,0 +1,40 @@
+import sqlite3
+import os
+
+from mitmproxy.io import protobuf
+
+
+class DbHandler:
+
+    """
+    This class is wrapping up connection to SQLITE DB.
+    """
+
+    def __init__(self, db_path, mode='load'):
+        if mode == 'write':
+            if os.path.isfile(db_path):
+                os.remove(db_path)
+        self.db_path = db_path
+        self._con = sqlite3.connect(self.db_path)
+        self._c = self._con.cursor()
+        self._create_db()
+
+    def _create_db(self):
+        with self._con:
+            self._con.execute('CREATE TABLE IF NOT EXISTS FLOWS('
+                              'id INTEGER PRIMARY KEY,'
+                              'pbuf_blob BLOB)')
```

Comment:
Next step: the schema here is going to become more complex as we add more tables, indexes and the like. We should pull this out, probably into its own separate .sql file.